lib/features/composer/presentation/composer_controller.dart

```diff
@@ -110,6 +112,22 @@ class ComposerController extends BaseController {
 
   void setSubjectEmail(String subject) => subjectEmail.value = subject;
 
+  selectTextStyleType(RichTextStyleType richTextStyleType){
+    if(listTextStyleApply.contains(richTextStyleType)){
+      listTextStyleApply.remove(richTextStyleType);
+    } else {
+      listTextStyleApply.add(richTextStyleType);
+    }
+  }
+
+  bool isTextStyleTypeSelected(RichTextStyleType richTextStyleType){
+    if(listTextStyleApply.contains(richTextStyleType)){
```

Comment:
You can abbreviate it like this: `return listTextStyleApply.contains(richTextStyleType)`